File: assignments/solutions/02-first-java/evenOdd.js
/**
 * Print input number is Even or Odd
 * 
 * @param {number} input
 * @returns {number}
 */

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

const evenOdd = (input) => {
    return input % 2 === 0 ? 'Even' : 'Odd';
}

console.log('Even Odd Number');
rl.question('Enter number: ', input => {
    console.log(`The number ${input} is ${evenOdd(input)}`);
    rl.close();
});

/**
 * Output:
 * 
 * Even Odd Number
 * Enter number: 3
 * The number 3 is Odd
 * 
 * Even Odd Number
 * Enter number: 3
 * The number 3 is Odd
 */

File: assignments/solutions/02-first-java/evenOdd.java
/*
Print input number is Even or Odd
*/

import java.util.Scanner;

public class evenOdd {
    public static void main (String arg[]) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Even Odd Number");
        System.out.print("Enter number: ");
        int input = scan.nextInt();
        if (input % 2 == 0)
            System.out.println("The number " + input + " is Even");
        else
            System.out.println("The number " + input + " is Odd");
    }
}

/*
Output:

Even Odd Number
Enter number: 2
The number 2 is Even

Even Odd Number
Enter number: 5
The number 5 is Odd
 */